### PR TITLE
Tabs to spaces

### PR DESCRIPTION
**Pull request by AndrewTHEManeri**
### Purpose of this PR
This PR converts the project from tabs to spaces - along with some other minor reformatting that the resharper applied.  I went through the files by hand and made sure the interface one-liners did not get expanded out (or rather, when they did I set them back)

### Testing status

Tested in a blank scene and the adventure scene with a variety of tools and workspaces.

### Technical risk

Low-medium.  This is all whitespace change, but it might confuse the next few merges that were developed off of the codebase when it was in tabs.

### Comments to reviewers

If you see some kind of spacing you want changed - just go ahead and make the change and commit it unless you _really_ think it merits discussion.

### CODE REVIEW

File: Scripts/Interfaces/FunctionalityInjection/ISetTooltipVisibility.cs [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
-		public static void HideTooltip(this ISetTooltipVisibility obj, ITooltip tooltip, bool persistent = false)
-		{
-			hideTooltip(tooltip, persistent);
-		}
-
-	}
+    /// <summary>
+    /// Provides access to the ability to show or hide a Tooltip
+    /// </summary>
+    public interface ISetTooltipVisibility {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/ISetTooltipVisibility.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesGameObjectLocking.cs [hunk 2]
```diff
@@ -4,36 +4,35 @@
...
-	/// <summary>
-	/// Get access to locking features
-	/// </summary>
-	public interface IUsesGameObjectLocking
-	{
-	}
+    /// <summary>
+    /// Get access to locking features
+    /// </summary>
+    public interface IUsesGameObjectLocking {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesGameObjectLocking.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesHandedRayOrigin.cs [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-	/// <summary>
-	/// Adds Node information to IUsesRayOrigin to determine which hand the tool is attached to
-	/// </summary>
-	public interface IUsesHandedRayOrigin : IUsesRayOrigin, IUsesNode
-	{
-	}
+    /// <summary>
+    /// Adds Node information to IUsesRayOrigin to determine which hand the tool is attached to
+    /// </summary>
+    public interface IUsesHandedRayOrigin : IUsesRayOrigin, IUsesNode {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesHandedRayOrigin.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesRaycastResults.cs [hunk 4]
```diff
@@ -4,26 +4,25 @@
...
-	/// <summary>
-	/// Gives decorated class access to hover/intersection detection
-	/// </summary>
-	public interface IUsesRaycastResults
-	{
-	}
+    /// <summary>
+    /// Gives decorated class access to hover/intersection detection
+    /// </summary>
+    public interface IUsesRaycastResults {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesRaycastResults.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesSnapping.cs [hunk 5]
```diff
@@ -4,61 +4,61 @@
...
-	/// <summary>
-	/// Gives decorated class the ability to use snapping
-	/// </summary>
-	public interface IUsesSnapping
-	{
-	}
+    /// <summary>
+    /// Gives decorated class the ability to use snapping
+    /// </summary>
+    public interface IUsesSnapping {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesSnapping.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesSpatialHash.cs [hunk 6]
```diff
@@ -4,35 +4,34 @@
...
-	/// <summary>
-	/// Make use of the spatial hash
-	/// </summary>
-	public interface IUsesSpatialHash
-	{
-	}
+    /// <summary>
+    /// Make use of the spatial hash
+    /// </summary>
+    public interface IUsesSpatialHash {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesSpatialHash.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesViewerBody.cs [hunk 7]
```diff
@@ -4,35 +4,34 @@
...
-	/// <summary>
-	/// Provides access to checks that can test against the viewer's body
-	/// </summary>
-	public interface IUsesViewerBody
-	{
-	}
+    /// <summary>
+    /// Provides access to checks that can test against the viewer's body
+    /// </summary>
+    public interface IUsesViewerBody {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesViewerBody.cs]: Brackets on individual lines

File: Scripts/Interfaces/FunctionalityInjection/IUsesViewerScale.cs [hunk 8]
```diff
@@ -3,32 +3,30 @@
 
 namespace UnityEditor.Experimental.EditorVR
 {
-	public interface IUsesViewerScale
-	{
-	}
+    public interface IUsesViewerScale {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/FunctionalityInjection/IUsesViewerScale.cs]: Brackets on individual lines

File: Scripts/Interfaces/IUsesCustomMenuOrigins.cs [hunk 9]
```diff
@@ -4,37 +4,36 @@
...
-	/// <summary>
-	/// Provides access to transform roots for custom menus
-	/// </summary>
-	public interface IUsesCustomMenuOrigins
-	{
-	}
+    /// <summary>
+    /// Provides access to transform roots for custom menus
+    /// </summary>
+    public interface IUsesCustomMenuOrigins {}
```
Comment (mtschoen-unity) [inline Scripts/Interfaces/IUsesCustomMenuOrigins.cs]: Brackets on individual lines

File: Scripts/UI/NumericInputField.cs [hunk 10]
```diff
@@ -4,276 +4,278 @@
...
+        {
+            ParseNumberField();
+        }
+
+        protected override void Space()
+        {
+            Return();
+        }
+
+        protected override void Shift() {}
```
Comment (mtschoen-unity) [inline Scripts/UI/NumericInputField.cs]: Brackets on individual lines

File: Scripts/UI/NumericInputField.cs [hunk 11]
```diff
@@ -4,276 +4,278 @@
...
+
+        protected override void CapsLock() {}
```
Comment (mtschoen-unity) [inline Scripts/UI/NumericInputField.cs]: Brackets on individual lines

File: Scripts/Utilities/CoroutineTween.cs [hunk 12]
```diff
@@ -6,680 +6,673 @@
...
+    //		tweenRunner.StartTween(positionTween);
+    //	}
+    //	
+    //	public void MoveObject(Vector3 position)
+    //	{
+    //		transform.position = position;
+    //	}
+    struct PositionTween : ITweenValue
+    {
+        public class PositionTweenCallback : UnityEvent<Vector3> {}
```
Comment (mtschoen-unity) [inline Scripts/Utilities/CoroutineTween.cs]: Brackets on individual lines

File: Scripts/Utilities/CoroutineTween.cs [hunk 13]
```diff
@@ -6,680 +6,673 @@
...
+
+        public bool ValidOnChangedTarget()
+        {
+            return m_OnChangedTarget != null;
+        }
+    }
+
+    struct RotationTween : ITweenValue
+    {
+        public class RotationTweenCallback : UnityEvent<Quaternion> {}
```
Comment (mtschoen-unity) [inline Scripts/Utilities/CoroutineTween.cs]: Brackets on individual lines

File: Scripts/Utilities/CoroutineTween.cs [hunk 14]
```diff
@@ -6,680 +6,673 @@
...
+
+        public bool ValidOnCompleteTarget()
+        {
+            return m_OnCompleteTarget != null;
+        }
+    }
+
+    struct ScaleTween : ITweenValue
+    {
+        public class ScaleTweenCallback : UnityEvent<Vector3> {}
```
Comment (mtschoen-unity) [inline Scripts/Utilities/CoroutineTween.cs]: Brackets on individual lines

File: Scripts/Utilities/CoroutineTween.cs [hunk 15]
```diff
@@ -6,680 +6,673 @@
...
+    struct ColorTween : ITweenValue
+    {
+        public enum ColorTweenMode
+        {
+            All,
+            RGB,
+            Alpha
+        }
+
+        public class ColorTweenCallback : UnityEvent<Color> {}
```
Comment (mtschoen-unity) [inline Scripts/Utilities/CoroutineTween.cs]: Brackets on individual lines

File: Scripts/Utilities/CoroutineTween.cs [hunk 16]
```diff
@@ -6,680 +6,673 @@
...
+            return m_OnChangedTarget != null;
+        }
+    }
+
+    // Float tween class, receives the
+    // TweenValue callback and then sets
+    // the value on the target.
+    struct FloatTween : ITweenValue
+    {
+        public class FloatTweenCallback : UnityEvent<float> {}
```
Comment (mtschoen-unity) [inline Scripts/Utilities/CoroutineTween.cs]: Brackets on individual lines

File: Scripts/Utilities/UnityBrandColorScheme.cs [hunk 17]
```diff
@@ -5,450 +5,474 @@
...
+
+        static Color s_DarkBlue;
+        static Color s_DarkBlueLight;
+
+        static Color s_Dark;
+        static Color s_Darker;
+        static Color s_Light;
+
+        // Unity Swatches 2016
+        public static Color red { get { return s_Red; } }
```
Comment (mtschoen-unity) [inline Scripts/Utilities/UnityBrandColorScheme.cs]: Should we get rid of the extra new lines?

File: Workspaces/Base/Workspace.cs [hunk 18]
```diff
@@ -9,306 +9,310 @@
...
+            m_OuterCollider.size = outerBounds.size;
+            m_OuterCollider.center = outerBounds.center;
+        }
+
+        protected virtual void OnDestroy()
+        {
+            destroyed(this);
+        }
+
+        protected virtual void OnBoundsChanged() {}
```
Comment (mtschoen-unity) [inline Workspaces/Base/Workspace.cs]: Brackets on individual lines

File: Workspaces/Common/Scripts/DraggableListItem.cs [hunk 19]
```diff
@@ -9,104 +9,99 @@
...
+        protected virtual void OnGrabDragStart(BaseHandle handle, HandleEventData eventData, Vector3 dragStart) {}
+
+        protected virtual void OnGrabDragging(BaseHandle handle, HandleEventData eventData, Vector3 dragStart) {}
+
+        protected virtual void OnDragEnded(BaseHandle handle, HandleEventData eventData)
+        {
+            m_DragObject = null;
+        }
+
+        protected virtual void OnMagnetizeEnded() {}
```
Comment (mtschoen-unity) [inline Workspaces/Common/Scripts/DraggableListItem.cs]: Brackets on individual lines

File: Workspaces/ConsoleWorkspace/ConsoleWorkspace.cs [hunk 20]
```diff
@@ -1,9 +1,10 @@
...
+#if UNITY_EDITOR
+
 namespace UnityEditor.Experimental.EditorVR.Workspaces
 {
-	[MainMenuItem("Console", "Workspaces", "View errors, warnings and other messages")]
-	sealed class ConsoleWorkspace : EditorWindowWorkspace
-	{
-	}
+    [MainMenuItem("Console", "Workspaces", "View errors, warnings and other messages")]
+    sealed class ConsoleWorkspace : EditorWindowWorkspace {}
```
Comment (mtschoen-unity) [inline Workspaces/ConsoleWorkspace/ConsoleWorkspace.cs]: Brackets on individual lines

File: Workspaces/InspectorWorkspace/Scripts/ListItems/InspectorListItem.cs [hunk 21]
```diff
@@ -10,369 +10,367 @@
...
+        {
+            return null;
+        }
+
+        protected virtual bool CanDropForFieldBlock(Transform fieldBlock, object dropObject)
+        {
+            return false;
+        }
+
+        protected virtual void ReceiveDropForFieldBlock(Transform fieldBlock, object dropObject) {}
```
Comment (mtschoen-unity) [inline Workspaces/InspectorWorkspace/Scripts/ListItems/InspectorListItem.cs]: Brackets on individual lines

File: Actions/BaseAction.cs [hunk 22]
```diff
@@ -3,20 +3,21 @@
...
-		Sprite m_Icon;
+        [SerializeField]
+        Sprite m_Icon;
 
-		public abstract void ExecuteAction();
-	}
+        public abstract void ExecuteAction();
+    }
 }
+
```
[the remark this answers is not in the document]
Comment (dunity) [reply Actions/BaseAction.cs]: Agreed. Not a big deal, but I feel similarly.

File: Scripts/Core/EditorVR.Rays.cs [hunk 23]
```diff
@@ -10,547 +10,549 @@
...
+                var selectionModule = obj as SelectionModule;
+                if (selectionModule)
+                {
+                    selectionModule.selected += SetLastSelectionRayOrigin; // when a selection occurs in the selection tool, call show in the alternate menu, allowing it to show/hide itself.
+                    selectionModule.getGroupRoot = GetGroupRoot;
+                    selectionModule.overrideSelectObject = OverrideSelectObject;
+                }
+            }
+
+            public void DisconnectInterface(object obj, Transform rayOrigin = null) {}
```
[the remark this answers is not in the document]
Comment (AndrewTHEManeri) [reply Scripts/Core/EditorVR.Rays.cs]: That applies to empty types (like classes)  - not methods.  In the class below there are one line functions listed

[the remark this answers is not in the document]
Comment (mtschoen-unity) [reply Scripts/Core/EditorVR.Rays.cs]: Huh. Not sure how I missed those methods. I'll go remove my comments on the methods, but there were some types (like interfaces) that this happened for.

File: Scripts/Utilities/UnityBrandColorScheme.cs (hunk 17 above)
Comment (AndrewTHEManeri) [reply Scripts/Utilities/UnityBrandColorScheme.cs]: yeah, you can shrink it down

**Comment by AndrewTHEManeri** [comment]
Manual merge successful, performing github merge

**Pull request merged**